scripts/svg.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const xml = require('xml2js');
+const builder = new xml.Builder({
+    renderOpts: {
+        pretty: false
+    },
+    headless: true
+});
+
+const svgImport = /import \* as *(?<variable>.*) from '(?<path>.*\.svg)';/i;
+
+const svgfolder = path.resolve('./svg/');
+
+const filenames = fs.readdirSync(svgfolder);
+
+let svgStrings = {};
+filenames.forEach(filename => {
+    const buffer = fs.readFileSync(path.join(svgfolder, filename));
+    xml.parseString(buffer.toString('utf8'), (err, result) => {
+        svgStrings[filename] = builder.buildObject(result);
+    });
+});
+
+const iconFile = path.resolve('./bin/es6/utils/icons.js');
+const iconFileContent = fs.readFileSync(iconFile, { encoding: 'utf8' });
+
+let newIconFileContent = [];
+const oldIconFileContent = iconFileContent.split('\n');
+oldIconFileContent.forEach(content => {
+    if (svgImport.test(content)) {
+        const matched = content.match(svgImport).groups;
+        const filename = path.basename(matched.path);
+        const changed = content.replace(svgImport, `const ${matched.variable} = \'${svgStrings[filename]}\';`);
```

Comment:
I would resolve the path of the svg file and read it from here with something like
```js
const matched = content.match(svgImport).groups;
const svgPath = path.resolve(iconFile,matched.path);
const svgString = fs.readFileSync(svgPath).toString();
try {
   const svgString = await  xml2js.parseStringPromise(
        svgString,
        {
            renderOpts: {
                pretty: false
            },
            headless: true
        }
    );
    const changed = content.replace(svgImport, `const ${matched.variable} = \'${svgString}\';`);
    newIconFileContent.push(changed);
} catch(e) {
    console.throw(`${svgPath} is not a valid XML`);
}
```

Reply:
the path is wrong you can't use it.